src/transport/tests/TestSessionManagerDispatch.cpp

```diff
@@ -100,19 +108,80 @@ struct MessageTestEntry theMessageTestVector[] = {
         .encryptedLength = 30,
         .privacyLength   = 30,
 
+        // NOTE: unicast message tests must use test key currently
+        .encryptKey = "\x5e\xde\xd2\x44\xe5\x53\x2b\x3c\xdc\x23\x40\x9d\xba\xd0\x52\xd2",
```

Comment:
Why?

Reply:
It's a limitation in the current implementation of `InjectPaseSessionWithTestKey`.  I can generate an issue to track and turn the NOTE: into a TODO(#issue):

Reply:
#22830